### PR TITLE
Fix off-by-one error with line ending character

### PR DESCRIPTION
**Pull request by floatdrop**
Just to test.

### CODE REVIEW

File: paste.c [hunk 1]
```diff
@@ -8,7 +8,7 @@ int wmain() {
 	HANDLE clip = GetClipboardData(CF_UNICODETEXT);
 	if (clip) {
 		_setmode(_fileno(stdout), _O_U8TEXT);
-		wprintf(L"%s\n", clip);
+		wprintf(L"%s", clip);
```
Comment (Qix-) [inline paste.c]: The format specifier is no longer necessary anymore. I would just use wputs.

Comment (floatdrop) [inline paste.c]: Yeah, there more problems around paste, than that. I think unicode/char-size is broken.

Comment (sindresorhus) [inline paste.c]: > I think unicode/char-size is broken.

@floatdrop Could you do a test to show that?

Comment (sindresorhus) [inline paste.c]: @Qix- Is it correct to not include the `\n`? I thought all output was supposed to include it.

Comment (Qix-) [inline paste.c]: Is it correct, yes.

Also, wide chars != unicode. Don't be fooled. Windows has kind of terrible support for unicode.

Comment (floatdrop) [inline paste.c]: @sindresorhus @Qix- I guess then test is wrong – copy reads file from stdin without additional new line, but paste will add one and files will not match.

**Pull request merged**